webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js

```diff
@@ -53,12 +63,19 @@ const ContentViewVersionContent = ({ cvId, versionId, cvVersion }) => {
         <a href={urlBuilder(`content_views/${cvId}#/versions/${versionId}/ansibleCollections`, '')}>{`${ansibleCollectionCount} Collections`}</a><br />
       </>
       }
+      <>
+        <span style={genericContentCountsStyle}>
+          {Object.keys(genericContentTypes).filter(typeCountKey => genericContentTypes[typeCountKey][0] > 0).map(typeCountKey => `${genericContentTypes[typeCountKey][0]} ${genericContentTypes[typeCountKey][1]}\n`)}
+        </span>
+      </>
       {(moduleStreamCount === 0 && debCount === 0 &&
         dockerManifestCount === 0 && dockerTagCount === 0 &&
         fileCount === 0 && ansibleCollectionCount === 0 &&
-        <TextContent>
-          <Text component={TextVariants.small}>{__('N/A')}</Text>
-        </TextContent>
+        Object.keys(genericContentTypes).map(typeCountKey =>
+          genericContentTypes[typeCountKey][0]).reduce((pv, cv) => pv + cv, 0) === 0 &&
```

Comment:
in `reduce((pv, cv) => pv + cv, 0)`, what is the "comma zero" for? Isn't that just adding 0 to the sum?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Reply:
removing the 0 doesn't seem to break anything for me

Reply:
It's the initial value, but looks like the default does the trick. I'll remove it.